recipes/_configure_indexes.rb

```diff
@@ -39,9 +39,10 @@
   daily_mb = hash.delete('_maxDailyDataSizeMB')
   padding = hash.delete('_dataSizePaddingPercent')
   default_config = config.fetch('default', {})
+  s2_enabled_index = hash.delete('_is_s2Index') || default_config['_is_s2Index']
```

Comment:
Can't we trigger off of either the remotePath setting or the storageType=remote configurations instead?  Those should already be provided in the databag to enable smartstore, so we shouldn't need a separate specific flag for it.

Reply:
Smart-store configuration for indexes.conf is deployed using an App, so remotePath and storageType=remote are not configured in data bag.

Reply:
So change where you configure it, or just add storageType=remote to the databag as well. The main point here is there are existing configurations for Splunk to indicate a remote store index.  We should use those to determine remote store rather than inventing custom configurations for it.